sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/zapwrapper/scan/ZapScanner.java

```diff
@@ -101,8 +106,10 @@ public void scan() throws ZapWrapperRuntimeException {
 
             /* After scan */
             generateZapReport();
+            informUserAboutAmountOfLogins();
             cleanUp();
```

Comment:
I would change the order inside the try block so if the first cleanUp fails for some reason the informUserAboutAmountOfLogins() is only called once inside the catch block. The catch block is fine. This way we ensure that the message is only sent once.
```suggestion
            cleanUp();
            informUserAboutAmountOfLogins();
```